### DISCUSSION TITLE
For "standard" group plans from a Payer, what should go into the Plan id, plan id type, and plant name in the json file?

### DISCUSSION THREAD

**Discussion by SatyaKomatineni** [CMSgov/price-transparency-guide#108] [Q&A] [answered]
Looking at the JSON

say I have a standard plan called "5060", a PPO plan, offered by "Payer1" (EIN: 444444) and it is offered to 1000 groups.

plan-id = (444444) EIN??? of the payer? if so xxxxxx (because no HIOS number)
plan-id-type = EIN (One of EIN  or HIOS) 
Plann-name= 5060
{ other data}

#another plan
plan-id = (444444) (same EIN)
plan-id-type = EIN (One of EIN  or HIOS) 
Plann-name= 5061 (assuming another standard plan)

Is it ok that plan ids are the same (because of EIN) for different plan names???

on the other hand if EIN is to represent the groups, then why am I repeating it 1000 times the same data?

Appreciate you looking into this.

Thanks
Satya

**Comment by shaselton-usds** [comment, marked as answer]
Hi @SatyaKomatineni 

Sorry for the delayed response -- I've converted your "Issue" to a "Discussion". It looks like your point is similar to what is being discussed here: https://github.com/CMSgov/price-transparency-guide/discussions/99. If you think it is, would you mind accepting this comment as the answer so we can consolidate the discussion?
